v0/destinations/blueshift/data/blueshiftGroupConfig.json

```
@@ -1,47 +1,85 @@
 [
   {
-    "destKey": "list_id",
+    "destKey": "group_id",
     "sourceKeys": "groupId",
     "sourceFromGenericMap": true,
-    "required": false
+    "required": true
   },
   {
-    "destKey": "identifier_value",
+    "destKey": "customer_id",
     "sourceKeys": "userId",
     "sourceFromGenericMap": true,
-    "required": true
+    "required": false
```

```suggestion
    "required": true
```
Isn't userId mandatory for mapping?

---

So sending a group call to blueshift without `customer_id` works?

---

How is the particular `user` identified then ?